app/src/main/res/layout-land/submitted_answer_item.xml

```diff
@@ -20,6 +20,7 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
+    android:id="@+id/submitted_answer_container"
```

Comment:
Should we add the same in its tablet related UI file?

Reply:
This is apparently already defined for tablets: https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/layout-sw600dp-land/submitted_answer_item.xml#L67. This PR is making the ID consistent across the layout files.